package/src/renderer/HostConfig.ts

```diff
@@ -192,9 +197,10 @@ export const skHostConfig: SkiaHostConfig = {
     if (shallowEq(prevProps, nextProps)) {
       return;
     }
-    const props = { ...nextProps };
+    const [props, reanimatedProps] = sanitizeReanimatedProps(nextProps);
     updatePayload.depMgr.unsubscribeNode(instance);
     instance.setProps(materialize(props));
+    bindReanimatedProps(container, node, reanimatedProps);
```

Comment:
Something looks off here with the variable names, can you check if this is correct?